x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/tests/notes.ts

```diff
@@ -384,14 +384,15 @@ export default function ({ getService }: FtrProviderContextWithSpaces) {
 
       // skipped https://github.com/elastic/kibana/issues/196896
       describe('@skipInServerless', () => {
-        it('should retrieve all notes that have been created by a specific user', async () => {
+        // TODO we need to figure out how to retrieve the uid of the current user in the test environment
+        it.skip('should retrieve all notes that have been created by a specific user', async () => {
```

Comment:
We should be able to enable it now, right?

Reply:
no actually now it's even more difficult as we need the uid of the user to test the api... I spent a few hours this afternoon trying to get this to work but couldn't find a way to retrieve that information. I also tried to create another user to be able to test the second skipped test but failed at that as well...

Reply:
Okay got it. Thanks for revert.